style.css

```diff
@@ -0,0 +1,145 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+/* Tool bar container */
+
+.toolbar {
+  width: 100%;
+  height: 92px;
+  position: fixed;
+  background-color: #3c3a39;
+  left: 0;
+  top: 0;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+header a {
+  width: 70px;
+  height: 24px;
+  color: #fff;
+  font-family: 'Roboto', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 24px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  top: 10px;
+  margin-left: 12px;
+}
+
+a:link {
+  color: white;
+}
+
+a:hover {
+  color: red;
+}
+
+a:visited {
+  color: #ff6b00;
+}
+
+header nav {
+  display: flex;
+  flex-direction: row;
+  align-items: flex-start;
+  padding: 8px;
+  position: absolute;
+  width: 40px;
+  height: 40px;
+  right: 24px;
+  top: calc(50% - 40px / 2);
+  border-radius: 8px;
+}
+
+/* section container */
+
+.section {
+  background-image: url(img/Group.png);
+  background-repeat: no-repeat;
+  background-position: 230px 70%;
+  background-color: #1c1a19;
+  padding: 120px 24px;
+  gap: 12px;
+  position: absolute;
+  width: 100%;
+  height: 590px;
+  left: 0;
+  top: 92px;
+  display: flex;
+  flex-direction: column;
+}
```

Comment:
- [ ] Please try to make the design as close as possible, As suggested by your previous code reviewer you did good work but still your design looks not good and I noticed that your icons are not visible any more.

![Screenshot from 2023-05-02 08-12-48](https://user-images.githubusercontent.com/66476329/235568960-0c84f623-c71f-4cda-8c49-0e7168d95aa5.png)